CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## 2.19.0-rc.0 / 2020-06-08
+
+* [FEATURE] Discovery: Added discovery support for Triton global zones. #7250
+* [FEATURE] TSDB: It now accepts callbacks to control series creation if you use TSDB as a package. #7159 
```

Comment:
This is an internal API which has no impact on the binary, it should not get mentioned.